recipe/bld.bat

```diff
@@ -13,6 +13,10 @@ cmake -G "NMake Makefiles JOM"              ^
     -DBUILD_SHARED_LIBS=on                  ^
     %SRC_DIR%
 
-jom install -j%CPU_COUNT%
+if defined USE_OPENMP (
```

Comment:
I assume this may be defined in both cases, but I'm going to wait a little longer for anaconda's cache to pick up the new build of flang.